synapse_data_warehouse/synapse_raw/tables/V2.20.0__create_accessrequirementsnapshots_table.sql

```diff
@@ -0,0 +1,87 @@
+USE SCHEMA {{database_name}}.synapse_raw; --noqa: JJ01,PRS,TMP
+USE WAREHOUSE COMPUTE_MEDIUM;
+
+CREATE OR ALTER TABLE ACCESSREQUIREMENTSNAPSHOTS (
```

Comment:
With a create or alter, wanted to get your thoughts on whether we could look into using `A__access_requirement_table` or something? This way we're always keeping the latest state of a table change in one script.

Reply:
I updated the PR so that the three tasks we need to accomplish are all included inseparate scripts:
- create the table in `R__create_accessrequirementsnapshots_table.sql`
- create the repeated update task in `R__accessrequirementsnapshots_append_task.sql`
- ingest the initial data we have in staging `V2.20.0__ingest_into_accessrequirementsnapshots.sql`